spec-0001.md

```diff
@@ -21,7 +23,11 @@ sp.linalg.eig(...)
 
 This was convenient: it had the simplicity of a flat namespace, but with the organization of a nested one.
 However, there was one drawback: importing submodules, especially large ones, introduced unacceptable slowdowns.
-To address the problem, most libraries stopped importing submodules and relied on documentation to tell users which submodules to import.
+
+For a while, SciPy had a lazy loading mechanism called `PackageLoader`.
+It was eventually dropped, because it failed frequently and in confusing ways—especially when used with interactive prompts.
+
+Thereafter, most libraries stopped importing submodules and relied on documentation to tell users which submodules to import.
```

Comment:
I didn't know about this - it's definitely a useful addition to the background :+1: